erica/erica_legacy/api/api.py

```diff
@@ -1,9 +1,7 @@
 from fastapi import APIRouter
 
 from erica.erica_legacy.api.v1.api_v1 import api_router_01
-from erica.erica_legacy.api.v2.api_v2 import api_router_02
 
 api_router = APIRouter()
 
 api_router.include_router(api_router_01)
-api_router.include_router(api_router_02)
```

Comment:
Depending on how we structure the main, we might have to revisit this (s. comment above)